### PR TITLE
Fix typo + remove unnecessary words_03-create.md

### PR DESCRIPTION
**Pull request by prosoitos**
- remove double word "then then" and make the sentence simpler.

### CODE REVIEW

File: _episodes/03-create.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ $ git status
 ~~~
 {: .bash}
 
-If you are using a different version of git than I am, then then the exact
+If you are using a different version of git, the exact
```
Comment (iglpdc) [inline _episodes/03-create.md]: Sounds good, but know you are it: could you please put quotes around git in this line? So git -> `git`. We use "Git" (capitalized) when talking about the protocol, a Git repository, or the system itself. We use `git` when talking about the implementation as a software program.

**Comment by iglpdc** [comment]
Changes look good, but if you have time could you add the change I mentioned? If you are busy, I'll merge it as is and open a new issue for the change.

File: _episodes/03-create.md (hunk 1 above)
Comment (prosoitos) [reply _episodes/03-create.md]: Will do!

**Comment by prosoitos** [comment]
Made suggested change.

**Comment by iglpdc** [comment]
Great, @prosoitos ! Thanks again for your contribution!

**Comment by prosoitos** [comment]
My pleasure :slightly_smiling_face:

**Pull request merged**